walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -28,7 +28,7 @@
 public final class FindMeetingQuery {
 
   /** Removes TimeRanges from times collection that are unavailable for the given attendees */
-  private ArrayList<TimeRange> removeConflictingTimes(ArrayList<TimeRange> times, Collection<String> attendees, Collection<Event> events, long duration){
+  private ArrayList<TimeRange> removeConflictingTimes(ArrayList<TimeRange> times, Collection<String> attendees, Collection<Event> events, long duration, boolean optimizing){
```

Comment:
Booleans should typically be named something like isX where X is the thing you're checking for (https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

So maybe something like isOptiminzingMeetingTimes would be good. it's a bit wordy but it tells the reader exactly what you mean with no extra effort required.

Reply:
Got it, thanks!